### ISSUE TITLE
Doesn't work with screen reader on chrome OS

### ISSUE THREAD

**Issue by BugsNash** [GoogleChromeLabs/text-app#360]
Several problems with using the screen reader on chrome OS (and presumably other screen readers), including keyboard traps, not able to read the CodeMirror text area at all (see [patch](https://chromium-review.googlesource.com/c/chromium/src/+/1020193) for same problem in chrome dev tools), and menu items that are not visible being selected by the screen reader.
